cachito/workers/pkg_managers/general_js.py

```diff
@@ -93,7 +94,23 @@ def download_dependencies(request_id, deps):
         for dep_batch in deps_batches:
             log.debug(f"Downloading the following npm dependencies: {', '.join(dep_batch)}")
             npm_pack_args = ["npm", "pack"] + dep_batch
-            run_cmd(npm_pack_args, run_params, "Failed to download the npm dependencies")
+            output = run_cmd(npm_pack_args, run_params, "Failed to download the npm dependencies")
+
+            # Move dependencies to their respective folders
+            # Make tuples out of dependency tarball and dep_identifier
+            # e.g., ('angular-animations-8.2.0.tgz', '@angular/animations@8.2.0')
+            tuples = list(zip(output.split(), dep_batch))
+            for dep_pair in tuples:
```

Comment:
Could you use a more descriptive name than `tuples` please?
```suggestion
            dep_pairs = list(zip(output.split(), dep_batch))
            for dep_pair in dep_pairs:
```

This would also be fine:
```suggestion
            for dep_pair in list(zip(output.split(), dep_batch)):
```